frameworks/g-cloud-9/questions/declaration/dunsNumber.yml

```diff
@@ -0,0 +1,10 @@
+question: What is your organisation’s DUNS number?
+type: text
+hint: >
+  The trading organisation applying to sell services through G-Cloud&nbsp;9 must have its own 9-digit DUNS number.
+  [Find out how to get a DUNS number.](http://www.dnb.co.uk/dandb-duns-number)
+
+validations:
+  -
```

Comment:
It's a bit pedantic but we've got a mix of:
```
validations:
  -
    name: 'foo'
    message: 'bar'
```
and
```
validations:
  - name: 'foo'
    message: 'bar'
```

Don't know if there is a difference? Not really a dealbreaker.